### ISSUE TITLE
Capture warnings in the log above the chart.

### ISSUE THREAD

**Issue by samussiah** [RhoInc/CAT#78]
hide warnings by default though

**Comment by jwildfire** [comment]
Yeah - this is clunky right now. We'll probably just want to refactor the current logging methods using something like [this](https://stackoverflow.com/questions/6604192/showing-console-errors-and-alerts-in-a-div-inside-the-page)
